docs/06-project-management/readme.md

```diff
@@ -0,0 +1,19 @@
+# Delivery Department
+
+Handbook for CivicActions' delivery department
+
+## Lead
+
+Elizabeth Raley serves as our Director of Professional Services and primary point of contact.
+
+## Index of Contents
+This folder contains the following subpages:
```

Comment:
Some pages have told more of a story that they embed links. i think we can work on that later if we want to enhance the readmes.